<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Qt3D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFVisualize3DWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>OpenGL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFVisualize3DGLWidget" name="widget2" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string> scene </string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>bounding box:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="chkBound">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>background color:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ColorComboBox" name="cmbBackground"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>ambient color:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ColorComboBox" name="cmbAmbient"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>current item:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cmbCurrent">
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string> geometry </string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>scene:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cmbScene">
            <item>
             <property name="text">
              <string>Test geometry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gauss*0.9*|sin|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|sin|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|sin|, cut</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string> color LUT </string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFLUTEditor" name="lutEditor">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>palette:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="JKQTPMathImageColorPaletteComboBox" name="cmbPalette"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string> transparency LUT </string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>alpha range:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="spinAlphaLower">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinAlphaUpper">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QFLUTEditor" name="lutAlphaEditor">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QFVisualize3DWidget</class>
   <extends>QWidget</extends>
   <header>qfvisualize3dwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JKQTPMathImageColorPaletteComboBox</class>
   <extends>QComboBox</extends>
   <header>../../../../LIB/trunk/qt/jkqtpimagetools.h</header>
  </customwidget>
  <customwidget>
   <class>QFLUTEditor</class>
   <extends>QFrame</extends>
   <header>qfluteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFVisualize3DGLWidget</class>
   <extends>QWidget</extends>
   <header>qfvisualize3dglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>../../../../LIB/trunk/qt/colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cmbScene</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update3DDataset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update3DDataset()</slot>
 </slots>
</ui>
